libr/cons/d/onedark

```diff
@@ -0,0 +1,62 @@
+#Inspired by onedark.vim
+e scr.color = 3
+ec func_var_type white
+ec func_var #61afef
+ec func_var_addr #4b5263
+ec b0x00 #c678dd
+ec b0xff #be5046
+ec prompt #d19a66
+ec fname #56b6c2
+ec label rgb:0f3
+ec math rgb:660
+ec bin #e5c07b
+ec btext #e5c07b
+ec call #e06c75
+ec jmp #c678dd
+ec cjmp #c678dd
+ec offset rgb:366
+ec comment #56b6c2
+ec usrcmt rgb:ff0
+ec push #61afef
+ec pop #61afef
+ec cmp rgb:060
+ec nop #abb2bf
+ec num #be5046
+ec reg #e5c07b
+ec fline rgb:fc0
+ec flow #98c379
+ec flow2 #98c379
+ec args #d19a66
+ec floc rgb:fc0
+ec flag #56b6c2
+ec label #56b6c2
+ec help #98c379
+ec input white
+ec invalid #e06c75
+ec crypto rgb:0000ff
+ec ret #be5046
+ec trap #e06c75
+ec swi #e06c75
+ec mov #98c379
+ec creg #e5c07b
+ec ai.read #98c379
+ec ai.write #61afef
+ec ai.exec #e06c75
+ec ai.seq rgb:ff00ff
+ec ai.ascii rgb:ffff00
+ec gui.cflow #d19a66
+ec gui.dataoffset #d19a66
+ec gui.background black
+ec gui.alt_background white
+ec gui.border black
+ec linehl #61afef
+ec graph.box #d19a66
+ec graph.box2 #61afef
+ec graph.box3 #c678dd
+ec graph.box4 rgb:000000
+ec graph.true #98c379
```

Comment:
better to use rgb: format instead of the # which may be confused with comments and fail if copypasted